packages/apollo-shared/src/ChangeManager/ChangeManager.ts

```diff
@@ -86,9 +99,81 @@ export class ChangeManager {
     }
   }
 
+  async changeThroughSocket(change: Change) {
```

Comment:
What is this function for? It looks like we should be able to `change.apply` and not have to worry about re-creating each change when one already exists.

Reply:
Fixed